Source/x11/XGServerWindow.m

```diff
@@ -4409,75 +4413,144 @@ - (void) freecursor: (void*) cid
   return depth;
 }
 
+/* This method assumes that we deal with one X11 screen - `defScreen`.
```

Comment:
Only in the XRandR case, for the fallback case multiple screens are possible.

Reply:
Theoretically, yes. Not working in practice. I've wrote you in my mail.

Reply:
I don't have a multi monitor setup, but others do. I would prefer to discuss this on the mailing list. I don't know whether it was possible to move a GNUstep window from one screen to another and how that was reflected in the windows data structure.

Reply:
I've made some additional testing (you can do it too using VirtualBox with "Monitor Count" set to 2 in VM preferences). I've made testing with `DISPLAY=:0.0`, `DISPLAY=:0`, with enable and disabled XRandR extension (`Option "RANR" "Disabled"` in xorg.conf). In all cases `ScreenCount` returns `1`.

Let me explain once more: GNUstep application can get only 1 (one) screen at a time. Screen identifier (XID) is determined at X11 initialization in backend by parsing DISPLAY environment variable and stored in `defScreen` (see _initXContext in XGServer.m). With current backend implementation (without XRandR support) there's no any chance to get multiple screens (Xlib screens).

What question/topic do we need to discuss in mailing list? I'm confused.

Reply:
I think that we left enough time for anybody to reply on the mailing list. As there was no response I withdraw my comment.